databricks/koalas/series.py

```diff
@@ -1065,6 +1065,8 @@ def name(self) -> Union[Any, Tuple]:
 
     @name.setter
     def name(self, name: Union[Any, Tuple]):
+        if not is_hashable(name):
+            raise TypeError("Series.name must be a hashable type")
```

Comment:
I don't think we need this if we check it in `rename`?

Reply:
Oh, rIght. I'll remove this. Thanks!